src/react-query/PersistQueryClientProvider.tsx

```diff
@@ -0,0 +1,60 @@
+// https://github.com/TanStack/query/blob/main/packages/react-query-persist-client/src/PersistQueryClientProvider.tsx
```

Comment:
Had issues where React Query was importing a different instance of the QueryClientProvider causing children to not consume the correct queryClient. I had a lot of trouble trying to figure out why, so I just copied the source.

Reply:
Do you happen to have an example of the errors you were getting? Seems odd.

But yeah if we need to go this route, mind just updating this comment to reference the permalink to this file? That way we'll get the snapshot of what the file looked like when you copied it.